tests/integration_tests/modules/test_users_groups.py

```diff
@@ -86,3 +89,30 @@ def test_user_root_in_secret(self, class_client):
         _, groups_str = output.split(":", maxsplit=1)
         groups = groups_str.split()
         assert "secret" in groups
+
+
+@pytest.mark.user_data(USER_DATA)
+def test_sudoers_includedir(client: IntegrationInstance):
+    """Ensure we don't add additional #includedir to sudoers.
```

Comment:
At the moment, the module docstring describes the test above this one; should that be moved to a class-level docstring?